README.md

```diff
@@ -16,17 +18,17 @@ English level B2 proven by Cambridge exam PET (Currently my level is ~C1, but I
 * Chatbots
 * And many more hackathons
 
-Just before COVID-19, I went to Saint Petersburg to participate in [Balt Konkurs](https://baltkonkurs.ru). Won 3rd-degree award and Youth jury laureate prize in "Computer Science'.
-Recently I had an internship at a company named Visyond.
-They are doing an excel for business people. It was a great experience☺️
-
+Just before COVID-19, I went to Saint Petersburg to participate in [Balt Konkurs](https://baltkonkurs.ru). Won 3rd-degree award and Youth jury laureate prize in "Computer Science'.  
```

Comment:
Trailing whitespace here and 2 lines after

Reply:
Actually, just learned this a few days ago, but if you put two spaces in a markdown, then they will be converted to '\n'

Reply:
Adding just the new line will leave a big space between the lines however, this won't

Reply:
Wow, cool feature.  Thanks!